game-core/src/main/java/games/strategy/engine/framework/startup/ui/posted/game/pbf/ForumPosterEditorViewModel.java

```diff
@@ -60,8 +82,14 @@ synchronized void setForumSelection(final String forumSelection) {
         "Forum selection is driven by a drop-down to ensure user can never set it to null, "
             + "if setting to null from a game properties, we default to the first available "
             + "selection entry, forum selection should never be null or empty.");
-    viewForumPostButtonEnabled = areFieldsValid();
-    testForumPostButtonEnabled = areFieldsValid();
+    forumUsername =
+        this.forumSelection.equals(NodeBbForumPoster.TRIPLEA_FORUM_DISPLAY_NAME)
+            ? ClientSetting.tripleaForumUsername.getValue().map(String::valueOf).orElse("")
+            : ClientSetting.aaForumUsername.getValue().map(String::valueOf).orElse("");
+    forumPassword =
+        this.forumSelection.equals(NodeBbForumPoster.TRIPLEA_FORUM_DISPLAY_NAME)
+            ? ClientSetting.tripleaForumPassword.getValue().orElse(new char[] {})
+            : ClientSetting.aaForumPassword.getValue().orElse(new char[] {});
```

Comment:
```suggestion
            ? ClientSetting.tripleaForumPassword.getValue().orElse(new char[0])
            : ClientSetting.aaForumPassword.getValue().orElse(new char[0]);
```

Reply:
Is this just personal preference @RoiEXLab ?

Reply:
Well yes, it's shorter and tbh I'm not really a fan of empty curly braces anywhere.
Just a weird thing of mine